sydent/http/servlets/replication.py

```diff
@@ -182,7 +187,9 @@ def _async_render_POST(self, request):
 
         # Process any invite token update
 
-        invite_updates = invite_tokens.get('updated', {})
+        invite_updates = OrderedDict(
```

Comment:
Err, we do `for updated_invite in invite_updates:` below, which kinda makes me feel like this might be a list rather than a dict? Or at least something is odd here?

Reply:
Yep, they're being sent as lists while everything else are dicts :man_facepalming: